reference/keywords.md

```diff
@@ -12,12 +12,13 @@ Introduced in: Clarity 1
 
 **description:**
 
-Returns the current block height of the Stacks blockchain as an uint
+Returns the current block height of the Stacks blockchain in Clarity 1 and 2 Upon activation of epoch 3.0, `block-height` will return the same value as `tenure-height`. In Clarity 3, `block-height` is removed and has been replaced with `stacks-block-height`.
```

Comment:
Lost a `.`:

```suggestion
Returns the current block height of the Stacks blockchain in Clarity 1 and 2. Upon activation of epoch 3.0, `block-height` will return the same value as `tenure-height`. In Clarity 3, `block-height` is removed and has been replaced with `stacks-block-height`.
```

Reply:
possible - had to manually clean up the json into markdown. commiting